tutorials/preparing-the-vm.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: Preparing the Virtual Machine
+---
+## Purpose
+This lesson describes the steps for configuring a virtual machine to run Kite example code on a Cloudera Quickstart VM.
+
+### Result
+Your VM is ready for you to install and run sample programs from the Kite SDK Examples project.
```

Comment:
These examples are modular and do not necessarily lead from one to the next. The progression is shown in a table of contents, not in the modules themselves.